src/ensembl/src/shared/state/genome/genomeActions.ts

```diff
@@ -51,6 +54,7 @@ export const fetchGenomeData = (
   ]);
 
   dispatch(ensureSpeciesIsCommitted(genomeId));
+  dispatch(ensureSpeciesIsEnabled(genomeId));
```

Comment:
I feel a bit uncomfortable about this. Logically, making sure that a species is enabled should be coupled with setting the active genome id, not with fetching data for genome.

Reply:
That's true. Do you know if there is a better option than calling this function separately from all the apps?

Reply:
Some tests are failing now.

Also, why did you add it in the react component for genome browser, but in a thunk action for entity viewer?

Reply:
Fixed the test.

>Also, why did you add it in the react component for genome browser, but in a thunk action for entity viewer?

I just thought of adding it next to where `fetchGenomeData` is called.